pkg/minikube/config/profile_test.go

```diff
@@ -73,24 +73,37 @@ func TestListProfiles(t *testing.T) {
 }
 
 func TestProfileNameValid(t *testing.T) {
-	var testCases = []struct {
-		name     string
-		expected bool
-	}{
-		{"meaningful_name", true},
-		{"meaningful_name@", false},
-		{"n_a_m_e_2", true},
-		{"n", false},
-		{"_name", false},
-		{"N__a.M--E12567", true},
-	}
-	for _, tt := range testCases {
-		got := ProfileNameValid(tt.name)
-		if got != tt.expected {
-			t.Errorf("expected ProfileNameValid(%s)=%t but got %t ", tt.name, tt.expected, got)
-		}
+	var testCases = map[string]bool{
+		"profile":             true,
+		"pro file":            false,
+		"pro-file":            true,
+		"pro-file-":           false,
+		"profile1":            true,
+		"pro-file1":           true,
+		"1st-profile":         true,
+		"1st-2nd-3rd-profile": true,
+		"-profile":            false,
+		"meaningful_name":     false,
+		"meaningful_name@":    false,
+		"n_a_m_e_2":           false,
+		"n":                   true,
+		"_name":               false,
+		"N__a.M--E12567":      false,
+		"1":                   true,
+		"12567":               true,
```

Comment:
Thank you for doing this PR for better readoablity I suggest moving all the false ones under each other

Reply:
85a641177 resolves